doc/features/aci-and-k8s-integration.md

```diff
@@ -0,0 +1,1139 @@
+# Cisco ACI and Kubernetes Integration
+
+- [Overview](#overview)
+- [Hardware Requirements](#hardware-requirements)
+- [Kubernetes Compatibility Matrix](#kubernetes-compatibility-matrix)
+- [Workflow for Kubernetes Integration](#workflow-for-kubernetes-integration)
+   * [Planning for Kubernetes Integration](#planning-for-kubernetes-integration)
+   * [Prerequisites for Integrating Kubernetes with Cisco ACI](#prerequisites-for-integrating-kubernetes-with-cisco-aci)
+   * [Provisioning Cisco ACI to Work with Kubernetes](#provisioning-cisco-aci-to-work-with-kubernetes)
+   * [Preparing the Kubernetes Nodes](#preparing-the-kubernetes-nodes)
+   * [Installing Kubernetes and Cisco ACI Containers](#installing-kubernetes-and-cisco-aci-containers)
+   * [Verifying the Kubernetes Integration](#verifying-the-kubernetes-integration)
+- [Unprovisioning Kubernetes from the ACI Fabric](#unprovisioning-kubernetes-from-the-aci-fabric)
+- [Uninstalling the CNI Plug-In](#uninstalling-the-cni-plug-in)
+- [Upgrade the CNI Plug-in](#upgrade-the-cni-plug-in)
+- [Using Policy](#using-policy)
+   * [Network Policy and EPGs](#network-policy-and-epgs)
+   * [Mapping to Cisco APIC](#mapping-to-cisco-apic)
+   * [Creating and Mapping an EPG](#creating-and-mapping-an-epg)
+   * [The acikubectl Command](#the-acikubectl-command)
+   * [Load Balancing External Services](#load-balancing-external-services)
+      + [Load Balancing External Services Across Multiple L3Out](#load-balancing-external-services-across-multiple-l3out)
+- [Draining a Node](#draining-a-node)
+- [Service Subnet Advertisement](#service-subnet-advertisement)
+   * [Configuring Service Subnet Advertisement](#configuring-service-subnet-advertisement)
+   * [Verifying Service Subnet Advertisement](#verifying-service-subnet-advertisement)
+- [Service Account Hardening](#service-account-hardening)
+   * [Checking the Current Administrative Privileges](#checking-the-current-administrative-privileges)
+   * [Modifying Administrative Account Permissions](#modifying-administrative-account-permissions)
+- [Troubleshooting Kubernetes Integration](#troubleshooting-kubernetes-integration)
+   * [Troubleshooting Checklist](#troubleshooting-checklist)
+   * [Troubleshooting Specific Problems](#troubleshooting-specific-problems)
+      + [Collecting and Exporting Logs](#collecting-and-exporting-logs)
+      + [Troubleshooting External Connectivity](#troubleshooting-external-connectivity)
+      + [Troubleshooting POD EPG Communication](#troubleshooting-pod-epg-communication)
+      + [Troubleshooting Endpoint Discovery](#troubleshooting-endpoint-discovery)
+      + [Troubleshooting Pod Health Check](#troubleshooting-pod-health-check)
+      + [Troubleshooting aci-containers-host](#troubleshooting-aci-containers-host)
+- [ACC provision features and configuration options](#acc-provision-features-and-configuration-options)
```

Comment:
Move this section prior to the "Troubleshooting section..."

Also, rename this section as "Feature and Configuration Details"